### DISCUSSION TITLE
Whether this plug-in has a size limit

### DISCUSSION THREAD

**Discussion by BsklWssy** [floccusaddon/floccus#1373] [Q&A] [answered]
May I ask if this is a size limit my bookmark more synchronous prompts this
![image](https://user-images.githubusercontent.com/32919368/232304683-ba65a87c-719e-4c11-b26e-a10e8a0b4d3c.png)
Then I deleted most of the bookmarks and it was successful so I wanted to ask if there was a size limit and I was using webdav

**Comment by marcelklehr** [comment, marked as answer]
mmh. There is no hard-coded size limit. This may instead be some kind of encoding issue
